lib/aur-srcver

```diff
@@ -3,6 +3,29 @@
 readonly argv0=srcver
 readonly PS4='+(${BASH_SOURCE}:${LINENO}): ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
 readonly startdir=$PWD
+makepkgargs=('--nobuild' '--nodeps' '--skipinteg')
+
+source /usr/share/makepkg/util/parseopts.sh
+opt_short=
+opt_long=('noprepare')
+opt_hidden=('dump-options')
+if ! parseopts "$opt_short" "${opt_long[@]}" "${opt_hidden[@]}" -- "$@"; then
+    echo "usage: $argv0 [--noprepare] <pkgbase> [<pkgbase> ...]" >&2
+    exit 1
+fi
+set -- "${OPTRET[@]}"
+unset opt_short opt_long opt_hidden OPTRET
+
+while true; do
+    case "$1" in
+        --noprepare)    makepkgargs+=("$1") ;;
```

Comment:
nitpick: elsewhere (`aur-sync`, `aur-build`) we use `makepkg_args` (with underscore)

Reply:
ok. will fix.